.gitignore

```diff
@@ -26,6 +26,9 @@ yarn-error.log*
 
 # local env files
 .env*.local
+.env
+.env.local
```

Comment:
.env.local is unnecessary here, it is covered with line 28
YOu should not need a .env file, since .env.local covers environment variables in Next.js projects